packages/lib/OgImages.tsx

```diff
@@ -70,14 +67,14 @@ export const constructMeetingImage = (
     // Joining a multiline string for readability.
   ].join("");
 
-  return encodeUri ? encodeURIComponent(url) : url;
```

Comment:
due to this double encoding, it results in 404, we already encode params here
<img width="550" alt="Screenshot 2024-11-27 at 5 18 55 AM" src="https://github.com/user-attachments/assets/eb33fa2a-da8a-4fb3-8b95-1351fcec4e28">